rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/policy/RestartDownNodesPolicy.java

```diff
@@ -108,7 +108,7 @@ private void checkDownNodesAndReacquireNodesIfNeeded() {
         try {
             long numberOfNodesToDeploy = this.nodeSource.getDownNodes()
                                                         .stream()
-                                                        .map(node -> node.getNodeInformation().getURL())
+                                                        .map(node -> node.getNodeURL())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)